### PR TITLE
Add Summer 2021 Track and Field fanart contest newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
SPORTS! 2.0 🌞

### CODE REVIEW

File: news/2021-07-29-summer-track-field-fanart-contest.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- Jumping (long jump, triple jump, high jump, pole vaulting)
+- Throwing (shot put, discus, hammers, javelins)
+
+This might seem a bit stiff for a fanart theme, but the astute among you will note that we included *shenanigans* in the theme. Sometimes races don't go to plan and people trip or things that shouldn't get thrown end up thrown, or marathon runners end up hopelessly lost where they shouldn't.
+
+Any mischief is fair game as long as it relates to track and field events somehow, so get creative! Yuzu floating over hurdles with a smug grin, Mocha tripping over every single one, Mani and Mari fighting over who gets to throw a discus first... the possibilities are more than you'd think!
+
+You've got **14 days** from the date of this post to bring your imagination to life, **but only after you've read all the rules below**.
+
+A top selection of the highest voted entries will make it to the main menu of the osu! game client (both stable and osu!lazer's preview builds) as our Summer 2021 Seasonal backgrounds, and nab their authors a fat lump (3 months) of osu! supporter. The top 3 entries will also receive a fancy profile badge of their own entry, too!
```
Comment (Walavouchey) [inline news/2021-07-29-summer-track-field-fanart-contest.md:32]:
```suggestion
A top selection of the highest voted entries will make it to the main menu of the osu! game client (both stable and osu!lazer's preview builds) as our Summer 2021 Seasonal backgrounds, and nab their authors a fat lump (3 months) of osu! supporter. The top 3 entries will also receive a fancy profile badge of their own entry, too!
```

File: news/2021-07-29-summer-track-field-fanart-contest.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- **All entries MUST be suitable for all-ages viewing.** Yes, athletics wear can sometimes hug one's contours. Whatever you draw our mascots wearing (or doing), please ensure that it is reasonable for an all-ages audience.
+- **Please do not include any logos or obvious references to the logos or branding of any global sporting events.** One such event happening in Tokyo right now tends to be very aggressive about where its logo gets used, so please keep things as generic as possible in this regard.
+- **SUBMIT ONLY YOUR OWN WORK. REDRAWING AND COMPOSITE WORKS ARE NOT ALLOWED.** Plagiarism sucks. Don't do it. Someone in the comments will call you out for it as has happened every other time someone's tried and they're going to do more damage to your reputation than us removing your entry from the voting will do. Use only your own work - brushes and the like are fine, but that's it! Don't redraw someone else's work.
```
Comment (Walavouchey) [inline news/2021-07-29-summer-track-field-fanart-contest.md:42]: long

```suggestion
- **SUBMIT ONLY YOUR OWN WORK. REDRAWING AND COMPOSITE WORKS ARE NOT ALLOWED.** Plagiarism sucks. Don't do it. Someone in the comments will call you out for it as has happened before and they're going to do more damage to your reputation than us removing your entry from the voting. Use only your own work — brushes and the like are fine, but that's it! Don't redraw someone else's work.
```

File: news/2021-07-29-summer-track-field-fanart-contest.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+You've got **14 days** from the date of this post to bring your imagination to life, **but only after you've read all the rules below**.
+
+A top selection of the highest voted entries will make it to the main menu of the osu! game client (both stable and osu!lazer's preview builds) as our Summer 2021 Seasonal backgrounds, and nab their authors a fat lump (3 months) of osu! supporter. The top 3 entries will also receive a fancy profile badge of their own entry, too!
+
+Time for the rules:
+
+## Rules
+
+- **By submitting your artwork to this contest, you grant us permission to use your artwork in the main menu of both clients (osu!stable and osu!lazer) should it win, and to use it in promotional material for both the game (osu!) and the contest (Summer 2021 Track and Field Fanart Contest).** You retain all rights to your creation and you can sell it and distribute it however you like - it is your work, after all. We're just saying here that if you submit your piece to this contest, you should be okay with us using it. If you aren't, please don't submit your entry!
```
Comment (Walavouchey) [inline news/2021-07-29-summer-track-field-fanart-contest.md:38]:
```suggestion
- **By submitting your artwork to this contest, you grant us permission to use your artwork in the main menu of both clients (osu!stable and osu!lazer) should it win, and to use it in promotional material for both the game (osu!) and the contest (Summer 2021 Track and Field Fanart Contest).** You retain all rights to your creation and you can sell it and distribute it however you like — it is your work, after all. We're just saying here that if you submit your piece to this contest, you should be okay with us using it. If you aren't, please don't submit your entry!
```

File: news/2021-07-29-summer-track-field-fanart-contest.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+- **ARTIST ATTRIBUTION (names, social media handles, etc) MUST BE LIMITED TO NO MORE THAN THREE LINES TOTAL AND CONFINED TO EITHER THE BOTTOM LEFT OR BOTTOM RIGHT CORNERS OF YOUR ENTRY.** Being proud of your work is good, just do it in a reasonably small spot of your work. Please understand that we're not very fond of anything ad-related in osu!, and attribution that is blaring enough can be considered as one.
+- **Entries MUST meet the resolution requirements for a main-menu piece - namely 2732x1536. THIS IS CHECKED BY A SCRIPT AND ENTRIES THAT DO NOT MATCH THIS REQUIREMENT WILL BE DISCARDED AUTOMATICALLY.** This is the #1 reason for entries not making it to voting phase. MAKE SURE YOUR ENTRY MEETS THE EXACT RESOLUTION REQUIREMENTS ABOVE. Check your work after you've exported it and BEFORE you submit it. Last time we had exactly NO incorrect resolution submissions which is fantastic, so let's keep the trend going!
```
Comment (Walavouchey) [inline news/2021-07-29-summer-track-field-fanart-contest.md:44]:
```suggestion
- **Entries MUST meet the resolution requirements for a main-menu piece — namely 2732x1536. THIS IS CHECKED BY A SCRIPT AND ENTRIES THAT DO NOT MATCH THIS REQUIREMENT WILL BE DISCARDED AUTOMATICALLY.** This is the #1 reason for entries not making it to voting phase. MAKE SURE YOUR ENTRY MEETS THE EXACT RESOLUTION REQUIREMENTS ABOVE. Check your work after you've exported it and BEFORE you submit it. Last time we had exactly NO incorrect resolution submissions which is fantastic, so let's keep the trend going!
```

**Review by Walavouchey — COMMENTED** [review]
looks alright

**Comment by Walavouchey** [comment]
following how it was [changed in the performance points newspost](https://github.com/ppy/osu-wiki/pull/5890/commits/43a2b977fcfa2b9a589f98a18be8eda818b46741), "osu!lazer" and "osu!stable" should be "osu!(lazer)" and "osu!(stable)", if i'm extrapolating correctly? also is this something that should be changed in the wiki going forward?

**Review by peppy — DISMISSED** [review]

**Comment by peppy** [comment]
Not sure you need to mention stable/lazer as separate apps. They aren't really. It'd be the same as listing cutting edge as well.

**Review by peppy — DISMISSED** [review]

File: news/2021-07-29-summer-track-field-fanart-contest.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+You've got **14 days** from the date of this post to bring your imagination to life, **but only after you've read all the rules below**.
+
+A top selection of the highest voted entries will make it to the main menu of the osu! game client (both in osu!(stable) and osu!(lazer) preview builds) as our Summer 2021 Seasonal backgrounds, and nab their authors a fat lump (3 months) of osu! supporter. The top 3 entries will also receive a fancy profile badge of their own entry, too!
+
+Time for the rules:
+
+## Rules
+
+- **By submitting your artwork to this contest, you grant us permission to use your artwork in the main menu of the osu! client) should it win, and to use it in promotional material for both the game (osu!) and the contest (Summer 2021 Track and Field Fanart Contest).** You retain all rights to your creation and you can sell it and distribute it however you like — it is your work, after all. We're just saying here that if you submit your piece to this contest, you should be okay with us using it. If you aren't, please don't submit your entry!
```
Comment (Walavouchey) [inline news/2021-07-29-summer-track-field-fanart-contest.md:38]:
```suggestion
- **By submitting your artwork to this contest, you grant us permission to use your artwork in the main menu of the osu! client should it win, and to use it in promotional material for both the game (osu!) and the contest (Summer 2021 Track and Field Fanart Contest).** You retain all rights to your creation and you can sell it and distribute it however you like — it is your work, after all. We're just saying here that if you submit your piece to this contest, you should be okay with us using it. If you aren't, please don't submit your entry!
```

**Review by peppy — APPROVED** [review]

**Pull request merged**